### DISCUSSION TITLE
有点可怕的鲨鱼希人硬要和我....

### DISCUSSION THREAD

**Discussion by xinxingjie** [share121/inter-knot#487] [其他]
今天和往常一样在去汀曼大师店里的路上，却发现一个睡着的鲨鱼希人，不小心把她叫醒后非要拉着我拍照，还打开我的手机让我设置成头像，嘴里还说着什么“你也不想做绳匠的事被发现吧?"之类的话，真的好可怕[狗头doge]
![艾莲光影](https://github.com/user-attachments/assets/cb9bb6e7-8896-4946-bb1f-4b8a57d989b4)

**Comment by warning-Kirov-reporting** [comment]
我直接抱住鲨希人的尾巴狂亲

**Comment by lbghhj** [comment]
跟楼上一样嘿嘿嘿（流口水）

**Comment by xinxingjie** [comment]
治安官在哪里，快把楼上这两个hentai抓走啊

**Comment by MiaoJiSir** [comment]
![微信图片_20240810094919](https://github.com/user-attachments/assets/64696b26-2ba0-498c-9147-07f1d301511a)
完成后，不要忘记到喵吉长官处领取奖励！

**Comment by XiaoFffeng** [comment]
喵吉长官，在绳网随意暴露身份是很危险的

**Comment by Elysia041** [comment]
好可爱～ ♪
